### ISSUE TITLE
DownloadAll in GUI drops files

### ISSUE THREAD

**Issue by donsizemore** [IQSS/dataverse#6454]
Cheryl and Mandy report that Harvard's Dataverse drops files from the GUI's Download-all-in-original-format function fairly regularly. As an example, Mandy reports that https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/REXZ8H&version=DRAFT
Mandy reports that the GUI will display the full number of datafiles selected for download, but the resulting .zip from the dataset above for instance lacks:

```
census_locality
Locations_2011
Results_2005
Results_2011
```

Mandy notes that the missing files seem to remain constant for each attempt at downloading all, but I'm opening an issue because they report that this problem happens often on Harvard Dataverse. She also reports that the problem became more common when Harvard enabled "tree-view."

**Comment by djbrooke** [comment]
Thanks @donsizemore for the report. We dropped the zip size limit on Harvard Dataverse in response to some recent instability, but we've made some performance improvements since, so I'll ask to have it increased again. I agree that it's not a good workflow and that people should be warned when they are about to download a zip that does not include all files. Right now people are see a note in the manifest.txt file, but that's after the fact and often missed. 

For my own info, what's the :ZipDownloadLimit set to in Odum's Dataverse?
